File: docs/code-examples/line_segments3d_simple.py
"""Log a simple set of line segments."""
import rerun as rr

rr.init("rerun_example_line_segments3d", spawn=True)

rr.log_line_segments(
    "simple",
    [
        [0, 0, 0],
        [0, 0, 1],
        [1, 0, 0],
        [1, 0, 1],
        [1, 1, 0],
        [1, 1, 1],
        [0, 1, 0],
        [0, 1, 1],
    ],
)

File: docs/code-examples/line_segments3d_simple.rs
//! Log a simple set of line segments.

fn main() -> Result<(), Box<dyn std::error::Error>> {
    let rec = rerun::RecordingStreamBuilder::new("rerun_example_line_segments3d").spawn()?;

    let points = [
        [0., 0., 0.],
        [0., 0., 1.],
        [1., 0., 0.],
        [1., 0., 1.],
        [1., 1., 0.],
        [1., 1., 1.],
        [0., 1., 0.],
        [0., 1., 1.],
    ];
    rec.log("segments", &rerun::LineStrips3D::new(points.chunks(2)))?;

    Ok(())
}
